<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminPage</class>
 <widget class="QMainWindow" name="AdminPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1290</width>
    <height>720</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0.458158, y1:0.451, x2:0.448522, y2:0.78, stop:0.00549451 rgba(43, 81, 169, 255), stop:1 rgba(60, 70, 169, 255));</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="White" native="true">
    <property name="geometry">
     <rect>
      <x>315</x>
      <y>0</y>
      <width>630</width>
      <height>720</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0.458158, y1:0.451, x2:0.448522, y2:0.78, stop:0.00549451 rgba(255, 255, 255, 255), stop:0.994505 rgba(255, 255, 255, 255));


background-color: qlineargradient(spread:pad, x1:0.492, y1:0, x2:0.508, y2:1, stop:0.010929 rgba(0, 0, 0, 130), stop:0.530055 rgba(59, 59, 59, 100));</string>
    </property>
    <widget class="QPushButton" name="PromoteToAdminButton">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>390</y>
       <width>211</width>
       <height>71</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: rgba(57, 68, 161,80);
    color: #ffdf00;
    font-family: 'Segoe UI', sans-serif;
    font-size: 15px;
    font-weight: bold;
    letter-spacing: 1px;
    text-transform: uppercase;
    padding: 13px 16px;
    border-radius: 20px;

    /* 3D outset-style border */
    border-style: solid;
    border-width: 5px;
    border-color: #ffffff #a0a0a0 #404040 #d0d0d0;  /* top, right, bottom, left */
}

QPushButton:hover {
	background-color: qlineargradient(spread:pad, x1:0.327839, y1:0.644, x2:1, y2:0.004, stop:0.010929 rgba(29, 59, 167, 255), stop:0.538462 rgba(58, 32, 142, 255));
}

QPushButton:pressed {
    /* Simulate button press by flipping light/shadow sides */
    border-color: #404040 #d0d0d0 #ffffff #a0a0a0;
	background-color: qlineargradient(spread:pad, x1:0.327839, y1:0.644, x2:1, y2:0.004, stop:0.010929 rgba(58, 32, 142, 255), stop:0.538462 rgba(29, 59, 167, 255));
    padding-top: 16px;
    padding-bottom: 10px;
}

QPushButton:disabled {
    background-color: #b0b0b0;
    color: #f0f0f0;
    border-color: #cccccc;
}
</string>
     </property>
     <property name="text">
      <string>Promote to admin</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BackButton">
     <property name="geometry">
      <rect>
       <x>203</x>
       <y>620</y>
       <width>211</width>
       <height>71</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: rgba(255, 0, 0, 100);
    color: #ffdf00;
    font-family: 'Segoe UI', sans-serif;
    font-size: 15px;
    font-weight: bold;
    letter-spacing: 1px;
    text-transform: uppercase;
    padding: 13px 16px;
    border-radius: 8px;

    /* 3D outset-style border */
    border-style: solid;
    border-width: 5px;
    border-color: #ffffff #a0a0a0 #404040 #d0d0d0;  /* top, right, bottom, left */
}

QPushButton:hover {
    background-color: rgba(255, 0, 0, 150)
}

QPushButton:pressed {
    /* Simulate button press by flipping light/shadow sides */
    border-color: #404040 #d0d0d0 #ffffff #a0a0a0;
    background-color: #780000;
    padding-top: 16px;
    padding-bottom: 10px;
}

QPushButton:disabled {
    background-color: #b0b0b0;
    color: #f0f0f0;
    border-color: #cccccc;
}
</string>
     </property>
     <property name="text">
      <string>Log Out</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>10</y>
       <width>151</width>
       <height>121</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent;border:none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>coin.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>153</x>
       <y>130</y>
       <width>321</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent; border:none</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Segoe UI','sans-serif'; font-size:36pt; font-weight:700; text-decoration: underline; color:#ffdf00;&quot;&gt;Admin Panel&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; text-decoration: underline;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_6">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>129</y>
       <width>351</width>
       <height>141</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent; border:none</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Segoe UI','sans-serif'; font-size:38pt; font-weight:700; text-decoration: underline; color:#000000;&quot;&gt;Admin&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ChangInfoButton">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>290</y>
       <width>211</width>
       <height>71</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: rgba(57, 68, 161,80);
    color: #ffdf00;
    font-family: 'Segoe UI', sans-serif;
    font-size: 15px;
    font-weight: bold;
    letter-spacing: 1px;
    text-transform: uppercase;
    padding: 13px 16px;
    border-radius: 20px;

    /* 3D outset-style border */
    border-style: solid;
    border-width: 5px;
    border-color: #ffffff #a0a0a0 #404040 #d0d0d0;  /* top, right, bottom, left */
}

QPushButton:hover {
	background-color: qlineargradient(spread:pad, x1:0.327839, y1:0.644, x2:1, y2:0.004, stop:0.010929 rgba(29, 59, 167, 255), stop:0.538462 rgba(58, 32, 142, 255));
}

QPushButton:pressed {
    /* Simulate button press by flipping light/shadow sides */
    border-color: #404040 #d0d0d0 #ffffff #a0a0a0;
	background-color: qlineargradient(spread:pad, x1:0.327839, y1:0.644, x2:1, y2:0.004, stop:0.010929 rgba(58, 32, 142, 255), stop:0.538462 rgba(29, 59, 167, 255));
    padding-top: 16px;
    padding-bottom: 10px;
}

QPushButton:disabled {
    background-color: #b0b0b0;
    color: #f0f0f0;
    border-color: #cccccc;
}
</string>
     </property>
     <property name="text">
      <string>Change Info</string>
     </property>
    </widget>
    <widget class="QPushButton" name="CurrentPriceButton">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>490</y>
       <width>211</width>
       <height>71</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: rgba(57, 68, 161,80);
    color: #ffdf00;
    font-family: 'Segoe UI', sans-serif;
    font-size: 15px;
    font-weight: bold;
    letter-spacing: 1px;
    text-transform: uppercase;
    padding: 13px 16px;
    border-radius: 20px;

    /* 3D outset-style border */
    border-style: solid;
    border-width: 5px;
    border-color: #ffffff #a0a0a0 #404040 #d0d0d0;  /* top, right, bottom, left */
}

QPushButton:hover {
	background-color: qlineargradient(spread:pad, x1:0.327839, y1:0.644, x2:1, y2:0.004, stop:0.010929 rgba(29, 59, 167, 255), stop:0.538462 rgba(58, 32, 142, 255));
}

QPushButton:pressed {
    /* Simulate button press by flipping light/shadow sides */
    border-color: #404040 #d0d0d0 #ffffff #a0a0a0;
	background-color: qlineargradient(spread:pad, x1:0.327839, y1:0.644, x2:1, y2:0.004, stop:0.010929 rgba(58, 32, 142, 255), stop:0.538462 rgba(29, 59, 167, 255));
    padding-top: 16px;
    padding-bottom: 10px;
}

QPushButton:disabled {
    background-color: #b0b0b0;
    color: #f0f0f0;
    border-color: #cccccc;
}
</string>
     </property>
     <property name="text">
      <string>See Current Prices</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_8">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>129</y>
       <width>351</width>
       <height>141</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent; border:none</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Segoe UI','sans-serif'; font-size:38pt; font-weight:700; text-decoration: underline; color:#000000;&quot;&gt;Panel&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="UserTradingRepoButton">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>390</y>
       <width>211</width>
       <height>71</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: rgba(57, 68, 161,80);
    color: #ffdf00;
    font-family: 'Segoe UI', sans-serif;
    font-size: 12px;
    font-weight: bold;
    letter-spacing: 1px;
    text-transform: uppercase;
    padding: 13px 16px;
    border-radius: 20px;

    /* 3D outset-style border */
    border-style: solid;
    border-width: 5px;
    border-color: #ffffff #a0a0a0 #404040 #d0d0d0;  /* top, right, bottom, left */
}

QPushButton:hover {
	background-color: qlineargradient(spread:pad, x1:0.327839, y1:0.644, x2:1, y2:0.004, stop:0.010929 rgba(29, 59, 167, 255), stop:0.538462 rgba(58, 32, 142, 255));
}

QPushButton:pressed {
    /* Simulate button press by flipping light/shadow sides */
    border-color: #404040 #d0d0d0 #ffffff #a0a0a0;
	background-color: qlineargradient(spread:pad, x1:0.327839, y1:0.644, x2:1, y2:0.004, stop:0.010929 rgba(58, 32, 142, 255), stop:0.538462 rgba(29, 59, 167, 255));
    padding-top: 16px;
    padding-bottom: 10px;
}

QPushButton:disabled {
    background-color: #b0b0b0;
    color: #f0f0f0;
    border-color: #cccccc;
}
</string>
     </property>
     <property name="text">
      <string>User Trading Reports</string>
     </property>
    </widget>
    <widget class="QPushButton" name="DeleteUserButton">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>490</y>
       <width>211</width>
       <height>71</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: rgba(57, 68, 161,80);
    color: #ffdf00;
    font-family: 'Segoe UI', sans-serif;
    font-size: 15px;
    font-weight: bold;
    letter-spacing: 1px;
    text-transform: uppercase;
    padding: 13px 16px;
    border-radius: 20px;

    /* 3D outset-style border */
    border-style: solid;
    border-width: 5px;
    border-color: #ffffff #a0a0a0 #404040 #d0d0d0;  /* top, right, bottom, left */
}

QPushButton:hover {
	background-color: qlineargradient(spread:pad, x1:0.327839, y1:0.644, x2:1, y2:0.004, stop:0.010929 rgba(29, 59, 167, 255), stop:0.538462 rgba(58, 32, 142, 255));
}

QPushButton:pressed {
    /* Simulate button press by flipping light/shadow sides */
    border-color: #404040 #d0d0d0 #ffffff #a0a0a0;
	background-color: qlineargradient(spread:pad, x1:0.327839, y1:0.644, x2:1, y2:0.004, stop:0.010929 rgba(58, 32, 142, 255), stop:0.538462 rgba(29, 59, 167, 255));
    padding-top: 16px;
    padding-bottom: 10px;
}

QPushButton:disabled {
    background-color: #b0b0b0;
    color: #f0f0f0;
    border-color: #cccccc;
}
</string>
     </property>
     <property name="text">
      <string>Delete User</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ViewUserButton">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>290</y>
       <width>211</width>
       <height>71</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: rgba(57, 68, 161,80);
    color: #ffdf00;
    font-family: 'Segoe UI', sans-serif;
    font-size: 15px;
    font-weight: bold;
    letter-spacing: 1px;
    text-transform: uppercase;
    padding: 13px 16px;
    border-radius: 20px;

    /* 3D outset-style border */
    border-style: solid;
    border-width: 5px;
    border-color: #ffffff #a0a0a0 #404040 #d0d0d0;  /* top, right, bottom, left */
}

QPushButton:hover {
	background-color: qlineargradient(spread:pad, x1:0.327839, y1:0.644, x2:1, y2:0.004, stop:0.010929 rgba(29, 59, 167, 255), stop:0.538462 rgba(58, 32, 142, 255));
}

QPushButton:pressed {
    /* Simulate button press by flipping light/shadow sides */
    border-color: #404040 #d0d0d0 #ffffff #a0a0a0;
	background-color: qlineargradient(spread:pad, x1:0.327839, y1:0.644, x2:1, y2:0.004, stop:0.010929 rgba(58, 32, 142, 255), stop:0.538462 rgba(29, 59, 167, 255));
    padding-top: 16px;
    padding-bottom: 10px;
}

QPushButton:disabled {
    background-color: #b0b0b0;
    color: #f0f0f0;
    border-color: #cccccc;
}
</string>
     </property>
     <property name="text">
      <string>View Users</string>
     </property>
    </widget>
    <zorder>textEdit_8</zorder>
    <zorder>textEdit_6</zorder>
    <zorder>PromoteToAdminButton</zorder>
    <zorder>BackButton</zorder>
    <zorder>label</zorder>
    <zorder>textEdit</zorder>
    <zorder>ChangInfoButton</zorder>
    <zorder>CurrentPriceButton</zorder>
    <zorder>UserTradingRepoButton</zorder>
    <zorder>DeleteUserButton</zorder>
    <zorder>ViewUserButton</zorder>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1290</width>
      <height>720</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;border:none</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>6073083.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>label_3</zorder>
   <zorder>White</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
